lib/view_model/cake_pay/cake_pay_cards_list_view_model.dart

```diff
@@ -152,20 +155,20 @@ abstract class CakePayCardsListViewModelBase with Store {
         onDemand: displayDenominationsCards);
 
     cakePayVendors = CakePayVendorList = newVendors;
-
     vendorsState = CakePayVendorLoadedState();
   }
 
   @action
   Future<void> fetchNextPage() async {
-    if (vendorsState is CakePayVendorLoadingState || !hasMoreDataToFetch || isLoadingNextPage)
+    if (vendorsState is CakePayVendorLoadingState || !hasMoreDataToFetch || isLoadingNextPage) {
       return;
+    }
 
     isLoadingNextPage = true;
     page++;
     try {
       var newVendors = await cakePayService.getVendors(
-          country: selectedCountry,
+          country: selectedCountry.title,
```

Comment:
shouldn't this be like the above call?
```suggestion
          country: Country.getCakePayName(selectedCountry),
```